proposals/lip-0058.md

```diff
@@ -82,7 +82,6 @@ As explained in [Section "Comparison to the Lisk-BFT paper" in LIP 0056][lip-005
 | Name | Description |
 |------|-------------|
 | `uint32be(x)` | For an integer `x` with `0 <= x < 2^32`, the function returns 4 bytes representing the big endian unsigned integer serialization of `x`. |
```

Comment:
Optional suggestion: We can even remove this function, since we kind of stopped using it from all other LIPs. Not a big deal, but since it is only used in 3 places over the whole LIP, we could just omit it (and even completely remove the table) and just replace it whenever used by `x.to_bytes(4,'big')`, see e.g., [notation in PoS LIP](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#snapshot-substore-1).

But as I said, this is a completely optional suggestion and it is okay to skip it.

Reply:
I would not make cosmetic changes to the LIP at this stage unless something is currently unclear or confusing.